docs/11.0/guides/upgrade-and-migration/migrating-from-10.0-to-11.0.md

```diff
@@ -44,6 +44,22 @@ Before, all of Handsontable's TypeScript definitions were kept in one huge file,
 
 Now, each module has its own TypeScript definitions file. They're all kept in a new `types` directory: [`./handsontable/types`](https://github.com/handsontable/handsontable/tree/master/handsontable/types).
 
+To import all of Handsontable's type definitions, simply load the `Handsontable` package, just like in JavaScript:
+
+```ts
+import Handsontable from 'handsontable';
+```
+
+To import an individual module's type definitions, import the module itself. For example:
+
+```ts
+import Handsontable from 'handsontable/base';
+
+import { registerPlugin, HiddenRows } from 'handsontable/plugins';
+
+const hiddenRows = Handsontable.plugins.HiddenRows;
+```
+
```

Comment:
The breaking change AFAIK mostly is introduced by editors' interfaces.

Before:
```js
class CustomEditor extends Handsontable.editors.BaseEditor implements Handsontable._editors.Base ()
```
After:
```js
class CustomEditor extends Handsontable.editors.BaseEditor implements Handsontable.editors.BaseEditor ()
```

Other changes just improve the typing. 

For example:
Before: 
```
new Handsontable.plugins.HiddenRows(hot); // TS Error (HiddenRows plugin is declared as interface) although it should be possible. In the runtime, this is a class.
```
After:
```
new Handsontable.plugins.HiddenRows(hot); // Ok (HiddenRows plugin is declared as class)
```

Reply:
@warpech @budnix - how about this? 54a5e16

Taking the occassion, I made a few other minor changes:
https://github.com/handsontable/handsontable/blob/docs/issue-8951/docs/11.0/guides/upgrade-and-migration/migrating-from-10.0-to-11.0.md

Reply:
Sounds good to me

Reply:
Sorry for late response. Please ping me on Slack if I don't respond for more than one day.

I believe that this can be simplified even more by merging the fragments "TypeScript definitions files" and "Importing all type definitions". I pushed my proposal on a separate branch. You can cherry-pick it if you agree: a558f49ce001a7cf588d960149418f69d8a1fc6c

Reply:
@warpech - ok, done: ab1ec7d

Reply:
Thank you!